### DISCUSSION TITLE
Cancel Request Issue

### DISCUSSION THREAD

**Discussion by Cammoy** [TanStack/query#797] [General]
I have seen the documentation but I am unsure how to call the cancel token before all request? can the example be extended to show that or can and example be provided here please?

**Comment by wolverineks** [comment]
can you be more specific? or perhaps provide an example of what you have so far?

**Reply by Cammoy** [reply]
@wolverineks, thanks for responding. What I mean is, if I perform a fetch using useQuery or similar and while the request is pending move to a new page, that request should be cancelled.

**Reply by wolverineks** [reply]
it would help if you were more specific about which part of the documentation is unclear, and/or provided a working example of what you have tried so far
